lib/strings/text.php

```diff
@@ -0,0 +1,53 @@
+<?php
+declare(strict_types=1);
+
+
+namespace jsys\types\strings;
+
+class Text implements TextInterface
+{
+    private $str;
+    public function __construct(string $str)
+    {
+        {
+            $this->str = $str;
+        }
+    }
+
+    public function isEmpty(string $str):bool
```

Comment:
what ???

Reply:
what are you trying to do here?
can you fix it ?